simp_le.py

```diff
@@ -223,6 +223,14 @@ def gen_csr(pkey, domains, sig_hash='sha256'):
     return req
 
 
+def get_le_tos_hash(le_uri):
+    """Returns up to date Let's Encrypt ToS hash"""
+    le_directory = requests.get(le_uri).json()
+    le_tos_uri = le_directory['meta']['terms-of-service']
```

Comment:
Would be nice to add some error checking here, so the user doesn't just get some nasty exception if the request fails or returns something unexpected.

Reply:
`sha256_of_uri_contents()` did not have error checking either, so I took the liberty to add some there too.

Reply:
Should probably also check response.ok; it won't raise an exception on non-success status codes.

Reply:
For `sha256_of_uri_contents()` you mean ?

On `get_le_tos_hash()`, anything that's not a 200 with valid JSON will be catched by `ValueError` when `json()` fails.

Also pylint complains about `ConnectionError`, is this a python 3 only subclass ? Should I use the parent class `OSError` instead ?

Reply:
Okay `sha256_of_uri_contents()` can't check for response status because the integration test except a 404 from boulder : https://github.com/letsencrypt/boulder/issues/1720